includes/api/class-wc-rest-products-controller.php

```diff
@@ -237,7 +237,7 @@ protected function prepare_objects_query( $request ) {
 		}
 
 		if ( ! empty( $tax_query ) ) {
-			$args['tax_query'] = $tax_query; // WPCS: slow query ok.
+			$args['tax_query'] = array_merge($tax_query, $args['tax_query']); // WPCS: slow query ok.
```

Comment:
- `$args['tax_query']` may not necessarily be set. This should check `$args['tax_query']` exists before doing the `array_merge`. [See the failing tests](https://travis-ci.org/woocommerce/woocommerce/jobs/408573093#L930-L1023)
- The WP style guide requires spaces between parentheses and their contents. [See the failing style check](https://travis-ci.org/woocommerce/woocommerce/jobs/408573093#L1036-L1042).

Reply:
Thanks @claudiulodro for your feedback! I have made your requested changes, hope it‘s fine now.

Reply:
Looks great! Thanks.